backend/src/routes/v0/tool/info.py

```
@@ -0,0 +1,77 @@
+                "mcp": [
+                    {
+                        k: v
+                        for k, v in tool.model_dump().items()
+                        if k not in ["func", "coroutine"]
+                    }
+                    for tool in tools
+                ]
+            },
+            status_code=status.HTTP_200_OK,
+        )
+    except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)